Number_types/include/CGAL/FPU.h

```diff
@@ -579,6 +612,23 @@ inline void force_ieee_double_precision()
 #endif
 }
 
+// Implementation (not part of the user interface)
+
+inline double IA_sqrt_up(double a) {
+  return IA_up(CGAL_BUG_SQRT(CGAL_IA_STOP_CPROP(a)));
+}
+
+inline double IA_sqrt_toward_zero(double d) {
+#ifdef CGAL_ALWAYS_ROUND_TO_NEAREST
+  return (d > 0.0) ? nextafter(std::sqrt(d), 0.) : 0.0;
+#else
+  FPU_set_cw(CGAL_FE_DOWNWARD);
+  double i = (d > 0.0) ? CGAL_BUG_SQRT(CGAL_IA_STOP_CPROP(d)) : 0.0;
```

Comment:
Did you lose CGAL_IA_FORCE_TO_DOUBLE somewhere?

Reply:
Yes, I think so.

I went over the substitutions manually and have included them in the code as temporary notes.

I found one CGAL_IA_FORCE_TO_DOUBLE that looks like it was omitted.